test/ca/mcgill/cs/jetuml/diagram/builder/TestClassDiagramBuilder.java

```diff
@@ -420,4 +402,143 @@ public void testCreateRemoveElementsOperationSingleNode()
 		operation.undo();
 		assertEquals(2, numberOfRootNodes());
 	}
+	
+	@Test
+	public void testCanAttachToPackageMultipleNodes()
+	{
+		ClassNode node1 = new ClassNode();

```

Comment:
Since the tests use different types of nodes, preinitialize all of the nodes before each test seems less efficient than only initializing the needed nodes in each test. I kept the original design and rename the variables to be more descriptive.